src/components/Button/Button.tsx

```diff
@@ -31,8 +32,8 @@ export function Button(props: ButtonProps) {
     <button
       data-testid="button"
       type="button"
+      className={`default-button px-4 py-3 ${buttonClass} ${customClass}`}
```

Comment:
```suggestion
      className={`default-button px-4 py-3 ${buttonClass}`}
```

Don't add a custom class, you've juste removed class `w-full` in the `global.css` file